railties/test/generators/app_generator_test.rb

```diff
@@ -668,6 +668,26 @@ def test_kamal_files_are_skipped_if_required
     assert_no_file ".env.erb"
   end
 
+  def test_inclusion_of_kamal_storage_volume
+    run_generator_and_bundler [destination_root]
+
+    assert_file "config/deploy.yml" do |content|
+      assert_match(%r{storage:/rails/storage}, content)
+    end
+  end
+
+  def test_kamal_storage_volume_is_skipped_if_required
+    run_generator_and_bundler [
+      destination_root,
+      "--skip-active-storage",
+      "--database=postgresql"
```

Comment:
Maybe there should be a test for `--skip-active-storage --database=sqlite3` or just `--skip-active-storage` since sqlite3 is the default?

Reply:
Ok, I added a test for this